src/models/bulletChart.js

```diff
@@ -21,9 +21,10 @@ nv.models.bulletChart = function() {
         , width = null
         , height = 55
         , tickFormat = null
-	, ticks = null
+	    , ticks = null
```

Comment:
There appears to be a tab there messing things up, should be all spaces to match the rest of the code.